src/lib/helper.js

```diff
@@ -18,7 +18,23 @@ const getItemsNamesFromDoc = (items) => {
   }, []);
 };
 
-const fromMilleToHours = (time) =>
-  (Date.now() - new Date(time)) / (1000 * 60 * 60);
+const getUTCNowInMilliSec = () => {
+  const now = new Date();
+  // get time now in milliseconds
+  const time = now.getTime();
+  // get timezone offset in minutes and convert it to milliseconds
+  // offset is how many milliseconds you should add to your time to convert to UTC
+  const offset = now.getTimezoneOffset() * 60000;
```

Comment:
Thank you for writing the comments here! ❤️

Reply:
Nice ! 👍

Reply:
I like that these date related functions are extracted out. Great in case you need to re-use it elsewhere.